assets/Javascript.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function () {
+    // array for topics
+    var topics = ["", "", ""]
+
+    // for loop function to make buttons for each item in this array
+    // set attributes for buttons to make on click animation easier
+
+
+
```

Comment:
Create a function that handles the process of clearing and displaying each gif category.  Also another that uses the api and adds a text that the user inputed to go through the api.